src/CounterLabel.tsx

```diff
@@ -4,16 +4,16 @@ import sx, {SxProp} from './sx'
 import {ComponentProps} from './utils/types'
 
 type StyledCounterLabelProps = {
-  scheme?: 'gray' | 'gray-light'
+  scheme?: 'primary' | 'secondary'
```

Comment:
This will need to be updated in `index.d.ts` as well.

Reply:
@colebemis I didn't find any references to this in `index.d.ts`

Reply:
This line should probably be `scheme?: 'primary' | 'secondary'` instead of `scheme?: string`:
https://github.com/primer/components/blob/main/index.d.ts#L236